drivers/wdm/audio/backpln/portcls/undoc.cpp

```diff
@@ -331,7 +323,7 @@ ASSERT(PropertyItem->Set);
     for(Index = 0; Index < SubDeviceDescriptor->FilterPropertySetCount; Index++)
     {
 
-		//RtlStringFromGUID(*SubDeviceDescriptor->FilterPropertySet[Index].Set, &GuidBuffer);
+                //RtlStringFromGUID(*SubDeviceDescriptor->FilterPropertySet[Index].Set, &GuidBuffer);
```

Comment:
do not touch that line. It was commented anyway. Your patch won't need that change.

Reply:
Fixed.